index.js

```diff
@@ -116,6 +117,7 @@ function do_bundle(file, opts){
                 if( opts.arithmetic ) {
                     jspm_input += ' ' + opts.arithmetic.trim();
                 }
+                jspm_input = jspm_input.replace(/\\/g, '/');
```

Comment:
`path.relative` returns a path depending on your platform. In my case on windows this was not recognized by the builder and caused it to fail. My guess is that it has something to do with the packages in the config.js not matching up because of the inverting of the slash character on windows.